tests/monitoring/metrics.go

```diff
@@ -67,6 +67,9 @@ var _ = Describe("[sig-monitoring]Metrics", decorators.SigMonitoring, func() {
 			"kubevirt_console_active_connections":                true,
 			"kubevirt_vmi_last_api_connection_timestamp_seconds": true,
 
+			// needs a snapshot - ignoring since already tested in VM snapshot metrics
+			"kubevirt_vmsnapshot_succeeded_timestamp_seconds": true,
```

Comment:
this metric is new, how is it already tested in `VM snapshot metrics`?
you can add a check in those tests to make sure the metric is correctly updated when the a VM snapshot is successfully created

Reply:
+1 created a new test.